### PR TITLE
use KBFS_QR identify behavior

### PR DESCRIPTION
**Pull request by songgao**
related: https://github.com/keybase/kbfs/pull/771

### CODE REVIEW

File: libkbfs/folder_block_manager.go [hunk 1]
```diff
@@ -944,10 +944,8 @@ func (fbm *folderBlockManager) finalizeReclamation(ctx context.Context,
 		gco.AddUnrefBlock(BlockPointer{ID: id})
 	}
 
-	// For now, pretend to be a rekey so the service suppresses
-	// popups.  TODO: add a more specific behavior type for QR.
 	ctx, err := makeExtendedIdentify(
-		ctx, keybase1.TLFIdentifyBehavior_KBFS_REKEY)
+		ctx, keybase1.TLFIdentifyBehavior_KBFS_QR)
```
Comment (strib) [inline libkbfs/folder_block_manager.go]: Please leave in some comment about how this asks the service to suppress popups for successful identifies.

Comment (songgao) [inline libkbfs/folder_block_manager.go]: sounds good! (although just making sure I understand correctly, this always suppress popups. The difference success/failure make is whether identify returns error. Sounds correct?)

Comment (strib) [inline libkbfs/folder_block_manager.go]: Ah yes, that's correct I think, sorry.

Comment (songgao) [inline libkbfs/folder_block_manager.go]: cool; thanks!

**Pull request merged**